docs/topics/http/views.txt

```diff
@@ -205,25 +205,26 @@ in a test view. For example::
 
 .. _async-views:
 
-Asynchronous views
-==================
+Async views
```

Comment:
Should we change this also in release notes?
`` `:ref:`Asynchronous views <async-views>` `` -> `` :ref:`async-views` ``

Reply:
I'm inclined to leave that because we're using the longer `Asynchronous` in the following bullet-points, so a rendered `Async` wouldn't be consistent. (Lemma: I think the longer form is appropriate here, at the beginning of the release notes.)

Reply:
:+1: